kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```
@@ -267,6 +267,7 @@ spec:
 
                     The feature can also be used to allow maximum time limits for long-running jobs which can delay node termination with preStop hooks.
                     If left undefined, the controller will wait indefinitely for pods to be drained.
+                  pattern: ^([0-9]+(\\.[0-9]+)?(s|m|h))+$
```

why do we need the middle bit? 
```suggestion
                  pattern: ^(([0-9]+(s|m|h))+)$
```